[debug] [2020-05-06T17:04:16.443Z] ----------------------------------------------------------------------
[debug] [2020-05-06T17:04:16.445Z] Command:       /usr/local/bin/firebase /Users/minhduc/.cache/firebase/tools/lib/node_modules/firebase-tools/lib/bin/firebase deploy
[debug] [2020-05-06T17:04:16.445Z] CLI Version:   7.9.0
[debug] [2020-05-06T17:04:16.445Z] Platform:      darwin
[debug] [2020-05-06T17:04:16.445Z] Node Version:  v10.15.3
[debug] [2020-05-06T17:04:16.447Z] Time:          Thu May 07 2020 00:04:16 GMT+0700 (Indochina Time)
[debug] [2020-05-06T17:04:16.447Z] ----------------------------------------------------------------------
[debug] [2020-05-06T17:04:16.447Z] 
[debug] [2020-05-06T17:04:16.459Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-05-06T17:04:16.459Z] > authorizing via signed-in user
[debug] [2020-05-06T17:04:16.459Z] [iam] checking project news-4c125 for permissions ["firebase.projects.get","firebasehosting.sites.update"]
[debug] [2020-05-06T17:04:16.461Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/news-4c125:testIamPermissions  

[debug] [2020-05-06T17:04:17.604Z] <<< HTTP RESPONSE 200
[debug] [2020-05-06T17:04:17.613Z] >>> HTTP REQUEST GET https://firebase.googleapis.com/v1beta1/projects/news-4c125  
 
[debug] [2020-05-06T17:04:18.850Z] <<< HTTP RESPONSE 200
[info] 
[info] === Deploying to 'news-4c125'...
[info] 
[info] i  deploying hosting
[debug] [2020-05-06T17:04:19.172Z] TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at validateString (internal/validators.js:125:11)
    at Object.resolve (path.js:1080:7)
    at resolveProjectPath (/Users/minhduc/.cache/firebase/tools/lib/node_modules/firebase-tools/lib/projectPath.js:6:17)
    at /Users/minhduc/.cache/firebase/tools/lib/node_modules/firebase-tools/lib/deploy/hosting/prepare.js:46:36
    at arrayEach (/Users/minhduc/.cache/firebase/tools/lib/node_modules/lodash/lodash.js:516:11)
    at Function.forEach (/Users/minhduc/.cache/firebase/tools/lib/node_modules/lodash/lodash.js:9342:14)
    at module.exports (/Users/minhduc/.cache/firebase/tools/lib/node_modules/firebase-tools/lib/deploy/hosting/prepare.js:29:7)
    at _chain (/Users/minhduc/.cache/firebase/tools/lib/node_modules/firebase-tools/lib/deploy/index.js:22:40)
    at /Users/minhduc/.cache/firebase/tools/lib/node_modules/firebase-tools/lib/deploy/index.js:63:16
    at process._tickCallback (internal/process/next_tick.js:68:7)
[error] 
[error] Error: An unexpected error has occurred.
